app/scripts/modules/core/src/pipeline/details/singleExecutionDetails.less

```diff
@@ -1,5 +1,3 @@
-.single-execution-details {
-  .checkbox {
-    padding: 0 1em;
-  }
+.single-execution-details__configure {
+  margin: 0 5px 0 1em;
```

Comment:
This ends up having the same effect as the replaced `.single-execution-details .checkbox` style rule?

Reply:
The aim is to have the right padding of 1em come off of the checkbox and instead have the configure button use it as 1em left margin. When the checkbox is alone it can sit flush on the right side and when the configure button is present it can maintain 1em spacing. What are you thoughts on it ? 😄 
![image](https://user-images.githubusercontent.com/2623242/53836138-37ab6a80-3f5d-11e9-9a2f-0871d1a62ac4.png)
![image](https://user-images.githubusercontent.com/2623242/53836194-527ddf00-3f5d-11e9-8676-71ea4f4a4c07.png)

Reply:
👍